cms/templates/studio_xblock_wrapper.html

```diff
@@ -128,14 +128,16 @@
     <div class="wrapper-xblock-message xblock-validation-messages" data-locator="${xblock.location | h}"/>
 % endif
 
-% if is_root or not xblock_url:
-    <article class="xblock-render">
-    ${content}
-    </article>
-% else:
-    <div class="xblock-message-area">
-    ${content}
-% endif
+% if xblock_context.get('show_preview', True):
```

Comment:
(Nit) As [implied by an earlier comment](https://github.com/edx/edx-platform/pull/6939/files#r24705629), it would be nicer if this could be a first-class template parameter, `% if show_preview`